src/lib/theme/base.ts

```diff
@@ -297,6 +297,20 @@ export const baseTheme = {
 		'11/12': '91.666667%',
 		'full': '100%',
 	},
+	aspectRatio: {
+		'1/1': 1 / 1,
+		'2/1': 2 / 1,
+		'3/1': 3 / 1,
+		'4/1': 4 / 1,
+		'5/1': 5 / 1,
+		'6/1': 6 / 1,
+		'7/1': 7 / 1,
+		'8/1': 8 / 1,
+		'9/1': 9 / 1,
+		'10/1': 10 / 1,
+		'11/1': 11 / 1,
+		'12/1': 12 / 1,
```

Comment:
How did we come up with these default values?

I'm asking because I didn't use the `aspectRatio` prop in RN before but I'd expect to see familiar values such as `16 / 9`. Other than `1/1`, I'm not sure how common the other values are used.

Reply:
I found them here [tailwindcss-aspect-ratio](https://github.com/tailwindlabs/tailwindcss-aspect-ratio) but now when I look at it, I am not sure why I didn't add them up to 16, will add it later. 
Also, I am thinking that maybe it makes sense to include the opposite range e.g.  `1 / 3`, `1 / 12` for when you want to make other side bigger.
I will also add `16 / 9` if you have any suggestions about more common values, let me know.

Reply:
> I found them here tailwindcss-aspect-ratio 

I see. But the syntax there is different. You would be using something like `aspect-w-16 aspect-h-9` instead of `aspect-16/9` if I got that correctly. 

Since we can't use a similar syntax we would need to rethink these values. 

I'd personally be fine with having only these for now and add more in the future if there is an actual demand:
* 1/1
* 2/1
* 3/2
* 4/3
* 16/9

Reply:
Sorry about my misinterpretation of values in `tailwindcss-aspect-ratio` didn't realize how that package actually workes. 🙈 
I changed default values as you suggested.